package.json

```diff
@@ -17,7 +37,15 @@
   },
   "homepage": "https://github.com/strothj/pirate-bartender#readme",
   "devDependencies": {
+    "chokidar-cli": "^1.2.0",
```

Comment:
chokidar is being used to rebuild the contents of the html and CSS. I'm still working through the webpack section. Later we use Webpack to do what we were doing with chokidar?